controls/srg_ctr/SRG-APP-000141-CTR-000315.yml

```diff
@@ -3,11 +3,12 @@ controls:
   levels:
   - medium
   title: {{{ full_name }}} must be configured with only essential configurations.
-  related_rules:
+  rules:
```

Comment:
One aspect of the usbgard rules is that at least two rounds of scan and remediations are needed for the rules to pass.
Because in the first round, the rule to install usbguard will be applied, but rules configuring it will result in not applicable.
In the second round, the rules configuring usbguard will fail, and their remediations will be applied.